articles/azure-monitor/app/asp-net-core.md

```diff
@@ -117,7 +117,7 @@ For Visual Studio for Mac use the [manual guidance](#enable-application-insights
 
 ### User secrets and other configuration providers
 
-If you want to store the instrumentation key in ASP.NET Core user secrets or retrieve it from another configuration provider, you can use the overload with a `Microsoft.Extensions.Configuration.IConfiguration` parameter. For example, `services.AddApplicationInsightsTelemetry(Configuration);`.
+If you want to store the instrumentation key in ASP.NET Core user secrets or retrieve it from another configuration provider, you can use the overload with a `Microsoft.Extensions.Configuration.IConfiguration` parameter. For example, `services.AddApplicationInsightsTelemetry(Configuration);`. Starting [Microsoft.ApplicationInsights.AspNet SDK](https://www.nuget.org/packages/Microsoft.ApplicationInsights.AspNetCore) version 2.15.0-beta3, calling `services.AddApplicationInsightsTelemetry()` will read instrumentation key from all [ASP.NET Core supported configuration providers](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1#cp) and there is no need to pass `Microsoft.Extensions.Configuration.IConfiguration` parameter explicitly. 
```

Comment:
Minor: Starting -> Starting from?

Reply:
Nit: remove "en-us" from the URL (I think the CI will catch this, too).

Reply:
Minor: put the new content into separate line?

Could the parameter `Microsoft.Extensions.Configuration.IConfiguration` be removed or deprecated from 2.15.0-beta3?

Reply:
Customer still have an option to use `IConfiguration`, we are not deprecating it. New changes allows customer to update the config without recompiling their app.